### ISSUE TITLE
[Suggestion] In-Game options tab

### ISSUE THREAD

**Issue by harald921** [remyroy/CDDA-Game-Launcher#12]
This is probably quite unecessary, but perhaps add a tab in the launcher where you can set key bindings and change the ingame options? For those who reinstall often, or for those who change settings often. Most likely more work than it's worth, but I'll put it here as a suggestion.

By the way, are you okay with me putting suggestions here like this, or should I post them somewhere else?

Keep up the great work!

**Comment by Malkeus** [comment]
This seems unnecessary. The launcher will copy your settings from version to version and changing them is handled in game.

**Comment by remyroy** [comment]
Putting suggestions here is fine.

That could be done but it would be a lot of work. There are so many options that would require so many UI for them. Eventually, that would be interesting to have it but I would focus on other features first.

**Comment by harald921** [comment]
@remyroy Yeah, I assume what would take the most time would be to actually make a good UI for all the settings, since I assume actually modifying the actual settings from outside the game isn't THAT hard? 

Either way, as you said - would be interesting but there are surely a lot of more important stuff that deserves focus atm. :)

**Comment by Nymall** [comment]
There is a in game UI for options, but it suffers from a LOT of the same clunk that other roguelikes suffer from. I'd like to be able to update a lot of settings such as standard worldgen parameters, options, debug, etc. without having to do the constant fight with the UI(a la Lazy Noob Pack for Dwarf Fortress).

**Comment by remyroy** [comment]
It seems reasonable. It will happen, eventually but I have other priorities first.

**Comment by Honcharuk** [comment]
Can I install the already downloaded file?

"Могу ли я установить уже скачанный архив?"
